src/components/Dashboard/RecentPastScheduledObs.vue

```diff
@@ -0,0 +1,68 @@
+<script setup>
+import { useRouter } from 'vue-router'
+import { useObsPortalDataStore } from '../../stores/obsPortalData.js'
+import { useConfigurationStore } from '../../stores/configuration.js'
+import { onMounted, ref } from 'vue'
+import { fetchApiCall } from '../../utils/api.js'
+import ObservationList from './ObservationList.vue'
+
+const router = useRouter()
+const obsPortalDataStore = useObsPortalDataStore()
+const completedObservations = obsPortalDataStore.completedObservations
```

Comment:
Not required to change, but it could be easier for someone reading this code later to see `store.completedObservations` used in line 18 instead of checking the variable name and then coming here to see it's the store. This is an optional/could-go-either-way thing though.